### PR TITLE
Adds spiders to the shapeshift menu!

### PR DESCRIPTION
**Pull request by ghost**
:cl:
add: Wizards may now shapeshift into viper spiders.
/:cl:

I think this is way better than just more random useless creatures, they have a purpose for shapeshifting the captain into a mouse but beyond that we don't need more. Alternatively, I think these are kinda neat! Thank you mister cobby for the suggestion.
closes #31689 because merge conflicts are annoying :(

### CODE REVIEW

**Review by ShizCalev — APPROVED** [review]

File: code/modules/spells/spell_types/shapeshift.dm [hunk 1]
```diff
@@ -18,6 +18,8 @@
 		/mob/living/simple_animal/pet/dog/corgi,\
 		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
 		/mob/living/simple_animal/bot/ed209,\
+		/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper,\
+		/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife,\
```
Comment (GunHog) [inline code/modules/spells/spell_types/shapeshift.dm]: PLAYER MIDWIFE nurse spiders? There is NO WAY this can end badly! :smiling_imp: :

Comment (Armhulen) [reply code/modules/spells/spell_types/shapeshift.dm]: <3

**Comment by jughu** [comment]
G O R R I L A

**Comment by ShizCalev** [comment]
@jughu just call gorillize() on them to turn them into a gorilla.

**Comment by KorPhaeron** [comment]
Self replicating on top of shapeshifts other benefits would make the spell too good for the cost, especially vs other minion oriented choices

**Comment by Armhulen** [comment]
having 40 health at any time playing wizard is a huge risk, two lasers or one shotgun blast is your end

**Comment by Armhulen** [comment]
byond that spawning hostile spiders isn't a fantastic way to stay alive as a wizard

**Comment by KorPhaeron** [comment]
I thought player controlled spiders created player controlled spiders who will follow a directive you set

**Comment by Dorsisdwarf** [comment]
Don't giant spiders have 200+hp?

**Comment by Doppeltongue** [comment]
I've taken several laser shots before as a player giant spider. Pretty sure I took a good amount of shotgun pellets and lived too.

**Comment by GunHog** [comment]
Midwives are 40 hp. The health values differ according to the type of spider. Nurse type spiders are weak in combat, and are meant to be protected by the other, more capable spiders.
https://github.com/tgstation/tgstation/blob/85d717350d5dda09a500e17da536128b30045e9c/code/modules/mob/living/simple_animal/hostile/giant_spider.dm#L182-L190

**Comment by Armhulen** [comment]
Oh right those dumb directives fuuuuck

Ok I'll remove midwife

**Comment by ExcessiveUseOfCobblestone** [comment]
just use the reg nurse instead then?

also fireball exists and it's a low cooldown stun that if it puts you on fire you're dead. Wizards can create equally if not worse rounds via summon guns/magic/events. Having a spider infestation sounds a lot more interesting than someone murdering with a saw or death bolt/charge combo, and it also puts people back into the round. Not to mention if the wizard morphs out then the spiders are just going to kill him because they know he ain't real and MUH VALIDS.

**Comment by Armhulen** [comment]
regular nuse also sets directives

**Comment by Armhulen** [comment]
i like a wizard having to dodge it's own spider armada but directives so

**Comment by JJRcop** [comment]
make a /wizard subtype that has a wizard hat on its sprite and doesn't grant set_directive

**Comment by Armhulen** [comment]
hm, okay

**Comment by Armhulen** [comment]
i need to save my tokens sorry

**Comment by ChangelingRain** [comment]
is this system working or *what*

**Comment by BlueNothing** [comment]
Hmm... no.

**Comment by Armhulen** [comment]
it works and you can merge it now it'll be good but if you wanna wait i can add nurse wizard spider

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-2). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**